### PR TITLE
2020 Changes

### PR DESCRIPTION
**Pull request by davish**
This is a first pass at changes to reflect the changes Labs has gone through in the past 5 years. Primarily, removing old mentions to Payroll and updating definitions and responsibilities. The Penn Labs Advisory Board membership was also updated to better reflect the reality of board meetings

Things we still need to add:
- [x] Add commitment to diversity and inclusion
- [x] Strengthen statement on Open Source

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -3,90 +3,199 @@ Penn Labs Constitution
...
-
-Though Penn Labs will partner with a variety of student groups on campus, funding for Penn
-Labs will remain independent of projects, leaving the group to work on the highest-impact products
-for student life, without regard to a particular student group’s ability to raise funding.
-Penn Labs cannot maintain a client-provider relationship with any student group. Proposals like
-payment per-project or conditional payment based on project completion are against the non-profit
-nature of the work and not considered potential funding sources.
+office for the school year.
 
 Penn Labs is open to receive donations beyond its base funding on an ad-hoc basis without
```
Comment (ArmaanT) [inline readme.md]: We should probably clarify that this doesn't prevent Labs from taking on contracted work.

File: readme.md [hunk 2]
```diff
@@ -3,90 +3,215 @@ Penn Labs Constitution
...
+and work to make the recruitment process as fair and inclusive as possible.
 
-The rate is consistent with rates at top research positions and the rates of computer science
-teaching assistants at Penn. Penn Labs may also, at its discretion, hire developers on a
-contract basis to fulfill specific needs not covered by the full time developers.
+### Open Source
 
-### Operations
+We believe that open source is the best way to develop cutting-edge applications while mantaining trust with
+the core user base. Open source allows us to quickly iterate using both user feedback and user code contribution.
```
Comment (davish) [inline readme.md]: not sure what "user code contribution" is... i guess that outside people can submit PRs? but that doesn't happen very often.

File: readme.md [hunk 3]
```diff
@@ -3,90 +3,215 @@ Penn Labs Constitution
...
+Perhaps more importantly, however, it allows our users to trust that we're handling their data responsibly. Specifically,
```
Comment (davish) [inline readme.md]: Maybe add something more philosophical about why it's important to give back to the open source community? We don't just use open source, we contribute sometimes too.

**Review by davish — COMMENTED** [review]
@pawalt some comments

**Pull request merged**